helm/templates/loki/configmap.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "loki.fullname" . }}
+  labels:
+    app: {{ template "loki.name" . }}
+    chart: {{ template "loki.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  loki.yaml: |
+    auth_enabled: {{ .Values.loki.configauth_enabled }}
+
+    server:
+      http_listen_port: {{ .Values.loki.service.port }}
+
+    ingester:
+      lifecycler:
+        address: {{ .Values.loki.config.ingester.lifecycler.address }}
```

Comment:
This line isn't necessary when running on k8s; loki will read the IP off of eth0.

Reply:
Good to know, removed